cors/handler_test.go

```diff
@@ -0,0 +1,89 @@
+package cors_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+
+	"github.com/SKF/go-enlight-middleware/cors"
+)
+
+func Test_Get(t *testing.T) {
+	options := []cors.Option{}
+
+	middleware := cors.New(options...)
+
+	request := httptest.NewRequest(http.MethodGet, "/", nil)
+
+	response := doRequest(request, middleware)
+	defer response.Body.Close()
+
+	allowOrigin, allowOriginExist := response.Header["Access-Control-Allow-Origin"]
```

Comment:
I would simply use `response.Header.Get("Access-Control-Allow-Origin")` since it is more idiomatic. It also removes the array handling below.

Reply:
Yes that was a lot better